config/default.yml

```diff
@@ -0,0 +1,5 @@
+Minitest/AssertNil:
```

Comment:
It seems that only `test` directory can be targeted.

```diff
+AllCops:
+  Include:
+    - '**/test/**/*'
+
 Minitest/AssertNil:
   Description: 'Check if your test uses `assert_nil` instead of `assert_equal(nil, something)`.'
```